doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-mortgages-example-proc.adoc

```diff
@@ -18,52 +18,68 @@ The package you select must contain all the required rule assets.
 . Click *Ok* to create and open the test scenario in the test scenarios designer.
```

Comment:
Not the regression but could we update the docs or my understanding of the app?
When I use Menu->Design->Projects I got smth similar to 
![image](https://user-images.githubusercontent.com/21653197/71288854-e9404e80-236b-11ea-9fc6-70c10ea9ca1d.png)
And I could not find the Mortgages there as well as three dots bcs I need to go to my space first.

Reply:
Done. Updated accordingly in the doc content.